### ISSUE TITLE
Unable to send_message

### ISSUE THREAD

**Issue by Benoit075** [tranhungt/tinder_pro#2]
First of all, thanks for this great Job ! 

I'm currently trying to send_message after a match but it's doesn't seem to work... 

``` javascript
tinder.like(result._id, function (err, res, body) {
                liked++;
                if (body.match) {
            console.log(result.name + ' ' + result._id);
            tinder.send_message(result._id, "Hello world!\n", function(err, res, body){
                console.log(err + ' ' + res + ' ' + body);
            });

                    matches++;
                }
}
```

Have you already encountered this problem ?

Thanks !

**Comment by tranhungt** [comment]
Hi there,
Thanks for writing in! I'll check it out.

**Comment by tranhungt** [comment]
@Benoit075 just curious, what errors are you getting? Where do you get the variable 'result' from? When you do result.name, result._id, do you mean res?

**Comment by tranhungt** [comment]
@Benoit075 Thanks again for pointing out the issue. I found a bug in the code where the user ID wasn't correctly put inside the request URL to send a message, and have released a patch, v1.0.2.

Please let me know if you're still having issues, and we can work through them.
